### PR TITLE
Update rates

### PR DESCRIPTION
**Pull request by nhakmiller**
### Background

The out of the box Correlation Rules were observed to drive high Snowflake warehouse utilization for customers & accounts that were previously not making heavy use of Scheduled or ad hoc queries. This PR adjusts up the `RateMinutes` value of several Correlation Rules in order to drive a lower overall warehouse utilization in Snowflake and reduce Snowflake spend.

### Changes

- Update `RateMinutes` to be a minimum of 60 minutes in all cases
- Updated `LookbackWindowMinutes` and `TimeoutMinutes` to be appropriately scaled to new `RateMinutes`

### Testing

- None

### CODE REVIEW

File: correlation_rules/aws_potentially_compromised_service_role.yml [hunk 1]
```diff
@@ -22,9 +22,9 @@ Detection:
         Match:
           - On: requestParameters.roleArn
     Schedule:
-      RateMinutes: 60
+      RateMinutes: 360
```
Comment (nhakmiller) [inline correlation_rules/aws_potentially_compromised_service_role.yml:25]: This was already at 60, and I wasn't sure if should downgrade the `LookbackWindowMinutes` to 90 or update the `RateMinutes` to something like `720` so I kind of met in the middle.

Comment (ben-githubs) [reply correlation_rules/aws_potentially_compromised_service_role.yml:25]: That seems fine!

**Review by ben-githubs — APPROVED** [review]
Looks good!

File: correlation_rules/aws_potentially_compromised_service_role.yml (hunk 1 above)
Comment (debugmiller) [reply correlation_rules/aws_potentially_compromised_service_role.yml:25]: Lookback shouldn't need to be larger than how late data arrives. for Cloudtrail I would 60 is more than enough?

Comment (debugmiller) [reply correlation_rules/aws_potentially_compromised_service_role.yml:25]: err 90

Comment (debugmiller) [reply correlation_rules/aws_potentially_compromised_service_role.yml:25]: so like `RateMinutes: 60`  `LookbackWindowMinutes: 90` like the others?

**Pull request merged**